art/attacks/evasion/fast_gradient.py

```diff
@@ -231,7 +231,13 @@ def generate(self, x: np.ndarray, y: Optional[np.ndarray] = None, **kwargs) -> n
                 # Use model predictions as correct outputs
                 logger.info("Using model predictions as correct labels for FGM.")
                 y = get_labels_np_array(self.estimator.predict(x, batch_size=self.batch_size))  # type: ignore
-            y = y / np.sum(y, axis=1, keepdims=True)
+
+            if self.estimator.nb_classes == 2 and y.shape[1] == 1:
+                axis = 0
```

Comment:
I'm not sure if we need to do anything for this case. Would `axis=0` sum over all samples in the binary case?